explorer-api/src/mix_nodes/http.rs

```diff
@@ -7,13 +8,83 @@ use rocket_okapi::openapi_get_routes_spec;
 use rocket_okapi::settings::OpenApiSettings;
 
 pub fn mix_nodes_make_default_routes(settings: &OpenApiSettings) -> (Vec<Route>, OpenApi) {
-    openapi_get_routes_spec![settings: list]
+    openapi_get_routes_spec![
+        settings: list,
+        list_active_set,
+        list_inactive_set,
+        list_standby_set,
+        summary
+    ]
 }
 
 #[openapi(tag = "mix_nodes")]
 #[get("/")]
 pub(crate) async fn list(
     state: &State<ExplorerApiStateContext>,
 ) -> Json<Vec<PrettyDetailedMixNodeBond>> {
-    Json(state.inner.mix_nodes.get_detailed_mixnodes().await)
+    Json(state.inner.mixnodes.get_detailed_mixnodes().await)
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/active-set/active")]
+pub(crate) async fn list_active_set(
+    state: &State<ExplorerApiStateContext>,
+) -> Json<Vec<PrettyDetailedMixNodeBond>> {
+    Json(get_mixnodes_by_status(
+        state.inner.mixnodes.get_detailed_mixnodes().await,
+        MixnodeStatus::Active,
+    ))
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/active-set/inactive")]
+pub(crate) async fn list_inactive_set(
+    state: &State<ExplorerApiStateContext>,
+) -> Json<Vec<PrettyDetailedMixNodeBond>> {
+    Json(get_mixnodes_by_status(
+        state.inner.mixnodes.get_detailed_mixnodes().await,
+        MixnodeStatus::Inactive,
+    ))
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/active-set/standby")]
+pub(crate) async fn list_standby_set(
+    state: &State<ExplorerApiStateContext>,
+) -> Json<Vec<PrettyDetailedMixNodeBond>> {
+    Json(get_mixnodes_by_status(
+        state.inner.mixnodes.get_detailed_mixnodes().await,
+        MixnodeStatus::Standby,
+    ))
+}
+
+#[openapi(tag = "mix_nodes")]
+#[get("/summary")]
+pub(crate) async fn summary(state: &State<ExplorerApiStateContext>) -> Json<MixNodeSummary> {
+    Json(get_mixnode_summary(state).await)
+}
+
+pub(crate) async fn get_mixnode_summary(state: &State<ExplorerApiStateContext>) -> MixNodeSummary {
+    let mixnodes = state.inner.mixnodes.get_detailed_mixnodes().await;
+    let active = get_mixnodes_by_status(mixnodes.clone(), MixnodeStatus::Active).len();
```

Comment:
I guess we could have sorted it into categories with a single loop iteration (rather than 3 of them), but this is so minor that unless this call is taking ages to compute, it's not really worth doing.

Reply:
Think it is ok for now